cookbook-react/src/recipes/selectors/selector-2/selector-2.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import Selector from './index';
+
+describe('Testing <Selector />', () => {
```

Comment:
```suggestion
describe('Selector', () => {
```
or just remove the describe